_docs/1.0.0/usage-schema.md

```diff
@@ -0,0 +1,618 @@
+| | |
+
+This is the value to be used in case the input is missing (and nulls are not
+allowed) or when the casting (*standardization*) fails. So better
+characterization would be, that it’s a *fallback value*.
+
+Should be noted, that this is the only _metadata_ key recognized that accepts the
+`null` value (written like that without quotes) next to string values.
+
+For more about the topic see chapter [Defaults](#defaults).
+
+### pattern
+
+| <sub><sup>Supported by types:</sup></sub> | <sub><sup>[Decimal](#decimal), [Long](#long), [Integer](#integer), [Short](#short), [Byte](#byte), [Double](#double), [Float](#float), [Timestamp](#struct), [Date](#timestamp)</sup></sub> |
+|-------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| | |
+
+The format the input adheres to. Mostly used for timestamp and date entries. But
+it can be also leveraged in case of numbers. Details for valid patterns are in
+the chapter [Parsing](#parsing).
+
+In case ``default`` value is also specified in _metadata_, it needs to adhere to
+the pattern.
+
+If [radix](#radix) is specified and differs from the default 10, `pattern` value
+will be ignored.
+
+### timezone
+
+| <sub><sup>Supported by types:</sup></sub> | <sub><sup>[Timestamp](#struct), [Date](#timestamp)</sup></sub> |
+|-------------------------------------------|----------------------------------------------------------------|
+| | |
+
+Time zone of the timestamp or date (for later it's generally not recommended). For details see the chapter
+[Parsing timestamps and dates](#parsing-timestamps-and-dates).
+
+In case the [`pattern`](#pattern) already includes information to recognize the time zone, the `timezone` entry in _metadata_ will
+be ignored. Namely if the pattern includes *"z"*, *"Z"* or *"X"* placeholder or *"epoch"*, *"epochmilli"*, *"epochmicro"*
+ and *"epochnano"* keywords.
+
+**NB!** Due to spark limitation, only time zone ids are accepted as valid values. To get the full list of supported time
+ zone denominators see the output of Java's
+[`TimeZone.getAvailableIDs()` function](https://docs.oracle.com/javase/8/docs/api/java/util/TimeZone.html#getAvailableIDs--).
+
+### decimal_separator
+
+| <sub><sup>Supported by types:</sup></sub> | <sub><sup>[Decimal](#decimal), [Long](#long), [Integer](#integer), [Short](#short), [Byte](#byte), [Double](#double), [Float](#float)</sup></sub> |
+|-------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| | |
+
+The character separating the integer and the fractional parts of the number.
+
+Seems pointless for whole numbers, but in case *"."* is used for example as
```

Comment:
```suggestion
For whole numbers which use *"."* as the [`grouping_separator`](#grouping_separator), the `decimal_separator` has to be redefined, to avoid conflict.
```